### ISSUE TITLE
Issuses about the accuracy

### ISSUE THREAD

**Issue by daifeng2016** [gaudetcj/IterativeSegmentation#2]
Hi,
    It is interesting to imitate PDEs using RNN for satellite classsification refinement. I have tested your code using my date set, which consists of 10000 256*256 satellite images. However, during the training phase, the loss keeps droping while the accuracy is always very low, which is below 0.01 in most cases, Hence, I am guessing about the problems: 1) Is the initial accuracy of CNN has great effect on the final RNN training process? 2) what is the prequisite to implement the result in your paper entitled "Recurrent Neural Networks to Correct Satellite Image Classifcation Maps"?  Could you provide more details.
   Thank you for your time.

**Comment by gaudetcj** [comment]
Hi,

I am not the author of the original paper, I only implemented it in Keras for practice.

The initial acc of the first pass CNN has to be pretty good for it to work on the second pass refinement since it uses the first pass predictions as the initial conditions of the second pass.
